components/stage/config/cloudify-stage.service

```diff
@@ -5,7 +5,8 @@ After=network-online.target
 
 [Service]
 TimeoutStartSec=0
-Restart=on-failure
+Restart=always
```

Comment:
need to do `always` because stage exits with a 0 error code on db connection failures